index.html

```diff
@@ -95,6 +95,19 @@
 
       <canvas id="canvas1" class="hidden-sm hidden-xs"></canvas>
       <div class="pycon2018">
+
+        <!-- share bar ribbon -->
+        <div class="share-bar-ribbon">
+         <ul class="list-vertical">
+            <li class="list-vertical__label">Share</li>
+
+            <li><a href="https://www.facebook.com/PyConIndia" target="_blank"><img src="img/fb.png" alt="image" class="list-vertical__icon"></a></li>
```

Comment:
Can we make these links a share button instead of just page links ?

Reply:
I mentioned the same in the pull request as well, since I had a look at pycon2017 website which has the same functionality. If someone can provide me the sharer links, I can add them here.

Reply:
I used http://www.sharelinkgenerator.com/ for generating the links. In case you want to change the content just head over there to generate the links with your content.

Reply:
thanks @ananyo2012, I'll update them today

Reply:
looks like we can only share URL with the Facebook post, and it needs a valid URL.
I have updated the URL for it.

Reply:
Also, what should be a better subject line for email share. I am keeping it *"PyCon India 2018 - Hyderabad"*. 
Let me know your suggestions.